### PR TITLE
[13.0][MIG] maintenance_team_hierarchy

### PR DESCRIPTION
**Pull request by jarroyomorales**
Migration to v13 of `maintenance_team_hierarchy`

@etobella @dalonsod

### CODE REVIEW

File: maintenance_team_hierarchy/models/maintenance_team.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            record.request_ids = self.env["maintenance.request"].search(
+                record._get_request_domains()
+            )
+
+    def _get_request_domains(self):
+        return [("maintenance_team_id", "child_of", self.id)]
+
+    @api.depends()
+    def _compute_new_todo_requests(self):
+        """This is the only way to remove the dependencies"""
```
Comment (etobella) [inline maintenance_team_hierarchy/models/maintenance_team.py]: Why do you need to remove the dependencies?

**Review by etobella — APPROVED** [review]

File: maintenance_team_hierarchy/models/maintenance_team.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    todo_request_ids = fields.One2many(compute="_compute_new_todo_requests")
+    todo_request_count = fields.Integer(compute="_compute_new_todo_requests")
+    todo_request_count_date = fields.Integer(compute="_compute_new_todo_requests")
+    todo_request_count_high_priority = fields.Integer(
+        compute="_compute_new_todo_requests"
+    )
+    todo_request_count_block = fields.Integer(compute="_compute_new_todo_requests")
+    todo_request_count_unscheduled = fields.Integer(
+        compute="_compute_new_todo_requests"
+    )
```
Comment (dalonsod) [inline maintenance_team_hierarchy/models/maintenance_team.py:30]: Is it possible to show on views these fields? If they are not shown, I don't know what their purpose is.

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comment. And, is it possible to add some demo data?

File: maintenance_team_hierarchy/models/maintenance_team.py (hunk 2 above)
Comment (jarroyomorales) [reply maintenance_team_hierarchy/models/maintenance_team.py:30]: These fields aren't new, we have just changed its compute function. They are already shown in the kanban view.

Comment (dalonsod) [reply maintenance_team_hierarchy/models/maintenance_team.py:30]: Reviewed on runbot, I don't see them in the kanvan view. Actually, there's no kanban view for `maintenance.team` in this addon

Comment (jarroyomorales) [reply maintenance_team_hierarchy/models/maintenance_team.py:30]: They are in the odoo core `maintenance` module. https://github.com/odoo/odoo/blob/13.0/addons/maintenance/views/maintenance_views.xml#L807

Comment (dalonsod) [reply maintenance_team_hierarchy/models/maintenance_team.py:30]: You're right, I was looking at kanban view of configuration menu, not dashboard one, sorry.

**Comment by etobella** [comment]
@dalonsod Can you review again once all the comments have been answered? :smile:

**Comment by dalonsod** [comment]
@etobella I asked for some demo data, if possible. @jarroyomorales would you finally include them? If not, I'll approve it anyway :D

**Comment by jarroyomorales** [comment]
@dalonsod @etobella Done!

**Review by dalonsod — APPROVED** [review]
👍

**Pull request merged**